db/migrations/0012.add_mivideo_tables.py

```diff
@@ -0,0 +1,18 @@
+'''
+Migrations for MiVideo tables
+'''
```

Comment:
Are you planning on editing this file in the future to add more tables? Asking because you're using plural tables here and in the file name. I think database changes need to be made incrementally in separate migrations for everything to work correctly.

Reply:
Yes.

Reply:
I don't think you can edit this in the future and have it do what you want. If the migration is marked as added in a database, I don't think (not 100 percent sure) any changes you make to the file will be applied. You can test this if you want, but I'm pretty sure you'll need to create a separate migrations.

Reply:
In that case, then yes, I'll tweak the docstring.